class/defaults.yml

```diff
@@ -45,3 +45,7 @@ parameters:
       - apiGroups: [extensions]
         resources: [networkpolicies]
         verbs: ['*']
+      # OpenShift specific
+      - apiGroups: [network.openshift.io]
+        resources: [egressnetworkpolicies]
+        verbs: ['*']
```

Comment:
I'm not sure if this is the correct way to add distribution-specific items here. Should we rather do this via parameters in hierarchy?
@corvus-ch  or @srueg WDYT?

Reply:
Yes, I'd suggest you add this in the hierarchy: https://git.vshn.net/syn/commodore-defaults/-/blob/master/distribution/openshift.yml

Reply:
Adding the following should do the trick (it gets merged with the rest of the array):
```
parameters:
  espejo:
    cluster_role_rules:
      - apiGroups: [network.openshift.io]
        resources: [egressnetworkpolicies]
        verbs: ['*']
```

Reply:
I agree with @srueg. Unless a component is clearly targeting a specific distribution (like the once having OpenShift in their name), nothing distribution specific should be part of a component.

Reply:
Works as described and a much cleaner solution. Thanks. Closing #14.